cmd/snap-seccomp-blacklist/snap-seccomp-blacklist.c

```diff
@@ -0,0 +1,215 @@
+#include <errno.h>
+#include <fcntl.h>
+#include <limits.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/ioctl.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#include <seccomp.h>
+
+__attribute__((format(printf, 1, 2))) static void showerr(const char *fmt, ...);
+
+static void showerr(const char *fmt, ...) {
+    va_list va;
+    va_start(va, fmt);
+    vfprintf(stderr, fmt, va);
+    fputc('\n', stderr);
+    va_end(va);
+}
+
+static int populate_filter(scmp_filter_ctx ctx, const uint32_t *arch_tags, size_t num_arch_tags) {
+    int sc_err;
+
+    /* If the native architecture is not one of the supported 64bit
+     * architectures listed above then remove it.
```

Comment:
I think with "listed above" you mean listed in `arch_tags`? Can you clarify this comment please?